src/schema/validator.js

```diff
@@ -205,17 +203,23 @@ export class SchemaValidator {
     this.allowedManifestVersionsRange =
       getManifestVersionsRange(validatorOptions);
 
-    const validator = ajv({
+    const validator = new Ajv({
+      strict: false,
       allErrors: true,
-      errorDataPath: 'property',
-      jsonPointers: true,
+      // include schema and data properties in error objects.
       verbose: true,
-      schemas: this.schemas,
-      schemaId: 'auto',
+      // TODO: do the following options have new replacements in ajv v8?
+      // what was the reason for setting these options in older ajv version?
+      //
+      // errorDataPath: 'property',
+      // jsonPointers: true,
```

Comment:
This configures the format of the `dataPath` (now `instancePath`). This is now the default according to https://github.com/ajv-validator/ajv/blob/0c40ebfa937d1d9cc01b4c5f06e7edfc4cfd5b00/docs/v6-to-v8-migration.md#removed-options

> jsonPointers: JSONPointers are used to report errors by default, jsPropertySyntax: true (deprecated) can be used if old format is needed.

Reply:
yeah, that is how I was reading that, but wanted to double-check it. I'm going to rip that off.